go.mod

```diff
@@ -0,0 +1,43 @@
+module github.com/iotexproject/uni-resolver-driver-did-io
+
+go 1.13
+
+require (
+	github.com/aristanetworks/goarista v0.0.0-20200521140103-6c3304613b30 // indirect
+	github.com/btcsuite/btcd v0.20.1-beta // indirect
+	github.com/edsrzf/mmap-go v1.0.0 // indirect
+	github.com/elastic/gosigar v0.10.5 // indirect
+	github.com/ethereum/go-ethereum v1.8.27
+	github.com/fjl/memsize v0.0.0-20190710130421-bcb5799ab5e5 // indirect
+	github.com/gballet/go-libpcsclite v0.0.0-20191108122812-4678299bea08 // indirect
+	github.com/go-openapi/errors v0.19.4
+	github.com/go-openapi/loads v0.19.5
+	github.com/go-openapi/runtime v0.19.15
+	github.com/go-openapi/spec v0.19.8
+	github.com/go-openapi/strfmt v0.19.5
+	github.com/go-openapi/swag v0.19.9
+	github.com/golang/protobuf v1.4.2 // indirect
+	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/huin/goupnp v1.0.0 // indirect
+	github.com/iotexproject/iotex-address v0.2.1
+	github.com/iotexproject/iotex-antenna-go/v2 v2.3.2
+	github.com/iotexproject/iotex-proto v0.3.0
+	github.com/jackpal/go-nat-pmp v1.0.2 // indirect
+	github.com/jessevdk/go-flags v1.4.0
+	github.com/karalabe/hid v1.0.0 // indirect
+	github.com/karalabe/usb v0.0.0-20191104083709-911d15fe12a9 // indirect
+	github.com/mattn/go-colorable v0.1.6 // indirect
+	github.com/olekukonko/tablewriter v0.0.4 // indirect
+	github.com/prometheus/tsdb v0.10.0 // indirect
+	github.com/rs/cors v1.7.0 // indirect
+	github.com/status-im/keycard-go v0.0.0-20200402102358-957c09536969 // indirect
+	github.com/steakknife/bloomfilter v0.0.0-20180922174646-6819c0d2a570 // indirect
+	github.com/steakknife/hamming v0.0.0-20180906055917-c99c65617cd3 // indirect
+	github.com/tyler-smith/go-bip39 v1.0.2 // indirect
+	github.com/wsddn/go-ecdh v0.0.0-20161211032359-48726bab9208 // indirect
+	golang.org/x/net v0.0.0-20200528225125-3c3fba18258b
+	gopkg.in/natefinch/npipe.v2 v2.0.0-20160621034901-c1b8fa8bdcce // indirect
+	gopkg.in/urfave/cli.v1 v1.20.0 // indirect
+)
+
+replace github.com/ethereum/go-ethereum v1.8.27 => github.com/iotexproject/go-ethereum v0.1.0
```

Comment:
i got this error when generating go.sum file:
```
go mod tidy
github.com/iotexproject/uni-resolver-driver-did-io/src/contract/IoTeXDID imports
	github.com/ethereum/go-ethereum/accounts/abi/bind tested by
	github.com/ethereum/go-ethereum/accounts/abi/bind.test imports
	github.com/ethereum/go-ethereum/accounts/abi/bind/backends imports
	github.com/ethereum/go-ethereum/eth/filters tested by
	github.com/ethereum/go-ethereum/eth/filters.test imports
	github.com/ethereum/go-ethereum/node imports
	github.com/prometheus/prometheus/util/flock: module github.com/prometheus/prometheus@latest found (v2.5.0+incompatible), but does not contain package github.com/prometheus/prometheus/util/flock
```

i think you will need to use `github.com/iotexproject/go-ethereum v0.2.0`